app/scripts/filters/resources.js

```diff
@@ -1274,15 +1274,36 @@ angular.module('openshiftConsole')
       });
     };
   })
-  .filter('scopeDetails', function() {
+  .filter('scopeDetails', function(sentenceCaseFilter) {
     var scopeMessages = {
       "Terminating": "Matches pods that have an active deadline.",
       "NotTerminating": "Matches pods that do not have an active deadline.",
       "BestEffort": "Matches pods that have best effort quality of service.",
       "NotBestEffort": "Matches pods that do not have best effort quality of service."
     };
     return function(scope) {
-      return scopeMessages[scope];
+      return scopeMessages[scope] || sentenceCaseFilter(scope);
+    };
+  })
+  .filter('isBestEffortPod', function(isNilFilter, usageValueFilter) {
```

Comment:
I put it in a filter because it felt like something we would want to use in the Pod view, but that was before the discussion upstream about the calculated QoS tier being added to the Pod status.  I can move it, we can always add a wrapper filter if we decide we need it later.